crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrSeaFloorDepth.cs

```diff
@@ -2,6 +2,7 @@
 
 // This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
 
+using System.Linq;
```

Comment:
Hi, just looking at this briefly, I can't see any reason why System.Linq would be needed in this file? Just going to test locally and will remove it if it isn't needed. Just giving a heads up here.

Reply:
For `Enumerable.Repeat`. I should replace it with something that doesn't use Linq since I think Linq can produce garbage.

Reply:
I've made a PR for this #451, yeah, we are making an array anyway, so the garbage isn't so much of a concern (as it's already being generated). It's a preference thing more than anything, but the little research I have done suggest that the Enumerable.Repeat is a tad slower as well anyway.